src/lib/cloudant.js

```diff
@@ -0,0 +1,336 @@
+/*
+  * Licensed Materials - Property of IBM
+  * (C) Copyright IBM Corp. 2016. All Rights Reserved.
+  * US Government Users Restricted Rights - Use, duplication or
+  * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+  */
+'use strict';
+
+const CloudantObj = require('cloudant');
+
+
+var env = null;
+
+/**
+ * Builds the options used to initialize with Cloudant (account/username/password)
+ * based on the environment variables.
+ *
+ * @return {object} [Object with api object with account/username/password fields]
+ */
+function buildEnv() {
+
```

Comment:
We should open a follow up issue to pull the env conditionally from VCAP if the service is bound to the container in Bluemix. @ycao56 can help with this.